app/screens/about/about.js

```diff
@@ -51,6 +51,14 @@ export default class About extends PureComponent {
         Linking.openURL(Config.MobileNoticeURL);
     };
 
+    handleTermsOfService = () => {
+        Linking.openURL(Config.TermsOfServiceURL);
```

Comment:
These values should come from the server's config. `assets/base/config.json` is actually something different that's included with the app at compile time. This component should already have the server config though since it's used to get the server build hash that's also used on this screen.